content/hi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,215 @@
+   उपलब्ध संसाधनों (सीपीयू, मेमोरी, आदि) में तैनात है।
+3. नोड का होस्टनाम और नेटवर्क पता प्राप्त करें।
+4. नोड के स्वास्थ्य का सत्यापन करना। यदि कोई नोड अनुत्तरदायी हो जाता है, तो यह नियंत्रक जाँच करता है
+   आपके क्लाउड प्रदाता की एपीआई यह देखने के लिए कि क्या सर्वर को निष्क्रिय/हटाया/समाप्त कर दिया गया है।
+   यदि नोड को क्लाउड से हटा दिया गया है, तो नियंत्रक आपके कुबेरनेट्स से नोड ऑब्जेक्ट को हटा देता है
+   झुंड।
+
+कुछ क्लाउड प्रदाता कार्यान्वयन इसे एक नोड नियंत्रक और एक अलग नोड में विभाजित करते हैं
+जीवनचक्र नियंत्रक।
+
+### मार्ग नियंत्रक
+
+रूट कंट्रोलर क्लाउड में रूट्स को कॉन्फ़िगर करने के लिए जिम्मेदार है
+उचित रूप से ताकि आपके कुबेरनेट्स में विभिन्न नोड्स पर कंटेनर हों
+क्लस्टर एक दूसरे के साथ संवाद कर सकते हैं।
+
+क्लाउड प्रदाता के आधार पर, रूट कंट्रोलर ब्लॉक भी आवंटित कर सकता है
+पॉड नेटवर्क के लिए IP पतों की संख्या।
+
+### सेवा नियंत्रक
+
+{{< glosary_tooltip text="Services" term_id="service" >}} क्लाउड के साथ एकीकृत
+इन्फ्रास्ट्रक्चर घटक जैसे कि प्रबंधित लोड बैलेंसर्स, आईपी पते, नेटवर्क
+पैकेट फ़िल्टरिंग, और लक्ष्य स्वास्थ्य जाँच। सेवा नियंत्रक आपके साथ इंटरैक्ट करता है
+क्लाउड प्रदाता के एपीआई लोड बैलेंसर्स और अन्य आधारभूत संरचना घटकों को स्थापित करने के लिए
+जब आप एक सेवा संसाधन की घोषणा करते हैं जिसकी उन्हें आवश्यकता होती है।
+
+## प्राधिकरण
+
+यह खंड क्लाउड कंट्रोलर मैनेजर के लिए आवश्यक एक्सेस को विभाजित करता है
+अपने कार्यों को करने के लिए विभिन्न एपीआई वस्तुओं पर।
+
+### नोड नियंत्रक {#प्राधिकरण-नोड-नियंत्रक}
+
+नोड नियंत्रक केवल नोड वस्तुओं के साथ काम करता है। इसके लिए पूर्ण पहुँच की आवश्यकता है
+नोड वस्तुओं को पढ़ने और संशोधित करने के लिए।
+
+`v1/Node`:
+
+- Get
+- List
+- Create
+- Update
+- Patch
+- Watch
+- Delete
+
+### मार्ग नियंत्रक {#प्राधिकरण-मार्ग-नियंत्रक}
+
+रूट कंट्रोलर नोड ऑब्जेक्ट के निर्माण को सुनता है और कॉन्फ़िगर करता है
+मार्ग उचित रूप से। इसके लिए आवश्यक है कि नोड वस्तुओं तक पहुंच प्राप्त करें।
+
+`v1/Node`:
+
+- Get
+
+### सेवा नियंत्रक {#प्राधिकरण-सेवा-नियंत्रक}
+
+सर्विस कंट्रोलर सर्विस ऑब्जेक्ट क्रिएट, अपडेट और डिलीट इवेंट्स को सुनता है और फिर उन सेवाओं के लिए एंडपॉइंट्स को उचित रूप से कॉन्फ़िगर करता है (एंडपॉइंटस्लाइस के लिए, क्यूब-कंट्रोलर-मैनेजर इन्हें डिमांड पर मैनेज करता है)।
+
+सेवाओं तक पहुँचने के लिए, इसे सूची और देखने की पहुँच की आवश्यकता होती है। सेवाओं को अपडेट करने के लिए, इसे पैच और अपडेट एक्सेस की आवश्यकता होती है।
+
+सेवाओं के लिए एंडपॉइंट संसाधन सेट अप करने के लिए, इसके लिए क्रिएट, लिस्ट, गेट, वॉच और अपडेट तक पहुंच की आवश्यकता होती है।
+
+`v1/Service`:
+
+- List
+- Get
+- Watch
+- Patch
+- Update
+
+### अन्य {#प्राधिकरण-विविध}
+
+क्लाउड कंट्रोलर मैनेजर के कोर के कार्यान्वयन के लिए इवेंट ऑब्जेक्ट बनाने के लिए एक्सेस की आवश्यकता होती है, और सुरक्षित संचालन सुनिश्चित करने के लिए सर्विस अकाउंट्स बनाने के लिए एक्सेस की आवश्यकता होती है।
+
+`v1/Event`:
+
+- Create
+- Patch
+- Update
+
+`v1/ServiceAccount`:
+
+- Create
+
+क्लाउड के लिए {{< glosary_tooltip term_id="rbac" text="RBAC" >}} ClusterRole
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.